pages/cart_page.py

```diff
@@ -0,0 +1,73 @@
+from playwright.sync_api import Page, expect
+
+class CartPage:
+    def __init__(self, page: Page):
+        self.page = page
+        self.subscription_text = self.page.locator("text=SUBSCRIPTION")
+        self.email_input = self.page.locator("#susbscribe_email")
+        self.subscribe_button = self.page.locator("#subscribe")
+        self.success_message = self.page.locator("#success-subscribe .alert-success")
+        self.cart_items_table = page.locator('#cart_info_table')  # Table containing cart items
+        self.cart_items = page.locator('#cart_info_table tbody tr')  # Rows for each product in the cart
+        self.product_names = self.cart_items.locator('.cart_description h4 a')  # Product names
+        self.product_prices = self.cart_items.locator('.cart_price p')  # Product prices
+        self.product_quantities = self.cart_items.locator('.cart_quantity button')  # Product quantities
+        self.product_totals = self.cart_items.locator('.cart_total .cart_total_price')  # Total price per product
+        self.cart_product_quantity = page.locator('tr#product-1 .cart_quantity button')
+        self.proceed_to_checkout_button = page.locator('a.btn.btn-default.check_out')
+        self.cart_page_heading = page.locator('.breadcrumb', has_text="Shopping Cart")
+        self.remove_product_button = page.locator('.cart_quantity_delete')
+
+
+    def verify_cart_page_displayed(self):
+        expect(self.cart_page_heading).to_be_visible()
+
+    def click_proceed_to_checkout(self):
+        self.proceed_to_checkout_button.click()
+
+    def verify_subscription_text_is_visible(self):
```

Comment:
@karenn32 Why is that assertion is here, but not in the spec file, since it has only single reference? Please, do not add all the assertions to the page files, only those, that repeated two or more times, like 'verify_cart_page_displayed' method.